sdk/identity/Azure.Identity/TROUBLESHOOTING.md

```diff
@@ -1,270 +1,222 @@
+|PowerShell isn’t installed.|No local installation of PowerShell was found.|Ensure that PowerShell is properly installed on the machine. Instructions for installing PowerShell can be found [here](tps://docs.microsoft.com/powershell/scripting/install/installing-powershell).|
+|Az.Account module >= 2.2.0 isn’t installed.|The Az.Account module needed for authentication in Azure PowerShell isn’t installed.|Install the latest Az.Account module. Installation instructions can be found [here](https://docs.microsoft.com/powershell/azure/install-az-ps).|
+|Please run 'Connect-AzAccount' to set up account.|No account is currently logged into Azure PowerShell.|<ul><li>Login to Azure PowerShell using the Connect-AzAccount command. More instructions for authenticating Azure PowerShell can be found [here](https://docs.microsoft.com/powershell/azure/authenticate-azureps)</li><li>Validate that Azure PowerShell can obtain tokens. See [below](#verifying-azure-powershell-can-obtain-tokens) for instructions.</li></ul>|
 
-#### Azure account not logged in.
+#### __Verifying Azure PowerShell can obtain tokens__
+You can manually verify that Azure PowerShell is properly authenticated, and can obtain tokens. First use the `Get-AzContext` command to verify the account which is currently logged in to the Azure CLI. 
 
-The `AzurePowerShellCredential` utilizes the current logged in Azure user in Azure Powershell to fetch an access token. You need to login to your account in Azure
-Powershell via `Connect-AzAccount` command. You can further read instructions
-to [Sign in with Azure Powershell](https://docs.microsoft.com/powershell/azure/authenticate-azureps). Once logged in try running the credential again.
+```
+PS C:\> Get-AzContext
 
-#### Deserialization error.
+Name                                     Account             SubscriptionName    Environment         TenantId
+----                                     -------             ----------------    -----------         --------
+Subscription1 (xxxxxxxx-xxxx-xxxx-xxx... test@outlook.com    Subscription1       AzureCloud          xxxxxxxx-x...
+```
 
-The `AzurePowerShellCredential` was able to retrieve a response from the Azure Powershell when attempting to get an access token but failed to parse that response.
-In your local powershell window, run the following command to ensure that Azure PowerShell is returning an access token in correct format.
+Once you've verified Azure PowerShell is using correct account, you can validate that it’s able to obtain tokens for this account.
 
-```pwsh
-Get-AzAccessToken -ResourceUrl "<Scopes-Url>"
+```bash
+Get-AzAccessToken -ResourceUrl "https://management.core.windows.net"
 ```
-
-In the event above command is not working properly, follow the instructions to resolve the Azure Powershell issue being faced and then try running the credential
-again.
+>Note that output of this command will contain a valid access token, and SHOULD NOT BE SHARED to avoid compromising account security.
```

Comment:
JS has added a note at the end. I think it would be nice if all languages have this note -

If this guide doesn't help you diagnose the errors you're experiencing, [open an issue](https://github.com/Azure/azure-sdk-for-js/issues). To contribute to the SDK, read the [contributing guide](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md).

Reply:
Added